lib/msal-node/package-lock.json

```diff
@@ -4550,6 +4545,2973 @@
       "optionalDependencies": {
         "commander": "^9.4.1"
       }
+    },
```

Comment:
So many changes in the `package-lock` file after bumping `msal-common` dep to the next version. That's odd.